sacc/data_types.py

```diff
@@ -32,7 +32,10 @@
     "xi_plus_im": ['theta'],
     "xi_minus_re": ['theta'],
     "xi_minus_im": ['theta'],
-    "count": []
+    "count": [],
+    "cluster_counts": [],
+    "cluster_mean_log_mass": [],
+    "cluster_shear": [],
```

Comment:
Could you add verbose versions of these?  I never wanted the concise versions to exist in the first place, Alonso made me do it.

Reply:
I used the same names for the verbose types. Should I remove them from the concise then?

Reply:
No, that's fine you can leave them in here.  That ship has sailed!